cloud_build/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright 2020 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Deploy a new flutter dashboard version to google cloud.
+
+cd app_dart
+gcloud app deploy --project flutter-dashboard --version "version-$1" --no-promote --no-stop-previous-version
```

Comment:
Let's deploy to flutter-dashboard-staging instead and create a manual trigger to deploy to prod.

Reply:
Done. Added separate build yaml files for staging and prod.